Plugin/Source/PluginProcessor.cpp

```diff
@@ -134,10 +134,15 @@ float ChowtapeModelAudioProcessor::calcLatencySamples() const noexcept
 
 bool ChowtapeModelAudioProcessor::isBusesLayoutSupported (const BusesLayout& layouts) const
 {
-    return ((! layouts.getMainInputChannelSet().isDiscreteLayout())
-            && (! layouts.getMainOutputChannelSet().isDiscreteLayout())
-            && (layouts.getMainInputChannelSet() == layouts.getMainOutputChannelSet())
-            && (! layouts.getMainInputChannelSet().isDisabled()));
+    if (wrapperType == juce::AudioProcessor::WrapperType::wrapperType_LV2) {
```

Comment:
If possible, would you mind running this through `clang-format`, just to keep with the formatting that the rest of the codebase uses?